saleor/account/migrations/0040_auto_20200415_0443.py

```diff
@@ -25,6 +25,9 @@ def change_extension_permission_to_plugin_permission(apps, schema_editor):
         user.user_permissions.remove(extension_permission)
         user.user_permissions.add(plugin_permission)
 
+    if extension_permission:
```

Comment:
could simply be `extension_permission.delete()` without the `if` as it doesn't raise an exception in case of empty QS.

Reply:
I'm using `.first()` method on QS, and it's return None for empty QS and this will raise an exception

Reply:
Aye, didn't see that, nvm.